### ISSUE TITLE
Translate to Italian

### ISSUE THREAD

**Issue by lewtun** [huggingface/course#45]
Hi there 👋

Let's translate the course to Italian so that the whole community can benefit from this resource 🌎!

Below are the chapters and files that need translating - let us know here if you'd like to translate any and we'll add your name to the list. Once you're finished, open a pull request and tag this issue by including `#issue-number` in the description, where `issue-number` is the number of this issue.

> 🙋 If you'd like others to help you with the translation, you can also post in our [forums](https://discuss.huggingface.co/c/course/20) or tag [@_lewtun](https://twitter.com/_lewtun) on Twitter to gain some visibility.

## Chapters

**0 - Setup**
- [x] [`1.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter0/1.mdx) @CaterinaBi

**1 - Transformer models**
- [x] [`1.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter1/1.mdx) @CaterinaBi
- [x] [`2.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter1/2.mdx) 
- [x] [`3.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter1/3.mdx) 
- [x] [`4.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter1/4.mdx) 
- [x] [`5.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter1/5.mdx) 
- [x] [`6.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter1/6.mdx) 
- [x] [`7.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter1/7.mdx) 
- [x] [`8.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter1/8.mdx) 
- [x] [`9.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter1/9.mdx) 
- [ ] [`10.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter1/10.mdx) 

**2 - Using 🤗 Transformers**
- [x] [`1.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter2/1.mdx)  @gdacciaro
- [x] [`2.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter2/2.mdx) 
- [ ] [`3.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter2/3.mdx) 
- [ ] [`4.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter2/4.mdx) 
- [ ] [`5.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter2/5.mdx) 
- [ ] [`6.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter2/6.mdx) 
- [ ] [`7.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter2/7.mdx) 
- [ ] [`8.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter2/8.mdx) 

**3 - Fine-tuning a pretrained model**
- [x] [`1.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter3/1.mdx) @sharkovsky
- [x] [`2.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter3/2.mdx) 
- [x] [`3.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter3/3.mdx) 
- [x] [`3_tf.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter3/3_tf.mdx) 
- [x] [`4.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter3/4.mdx) 
- [x] [`5.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter3/5.mdx) 
- [x] [`6.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter3/6.mdx) 

**4 - Sharing models and tokenizers**
- [x] [`1.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter4/1.mdx) [@ClonedOne](https://github.com/ClonedOne)
- [x] [`2.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter4/2.mdx)
- [x] [`3.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter4/3.mdx)
- [x] [`4.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter4/4.mdx)
- [x] [`5.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter4/5.mdx)
- [x] [`6.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter4/6.mdx)

**5 - The 🤗 Datasets library**
- [x] [`1.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter5/1.mdx) [@Nolanogenn](https://github.com/Nolanogenn)
- [x] [`2.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter5/2.mdx)
- [x] [`3.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter5/3.mdx)
- [x] [`4.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter5/4.mdx)
- [x] [`5.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter5/5.mdx)
- [x] [`6.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter5/6.mdx)
- [x] [`7.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter5/7.mdx)
- [x] [`8.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter5/8.mdx)

**6 -  The 🤗 Tokenizers library**
- [ ] [`1.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter6/1.mdx) [@davidemastricci](https://github.com/davidemastricci)
- [ ] [`2.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter6/2.mdx)
- [ ] [`3.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter6/3.mdx)
- [ ] [`3b.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter6/3b.mdx)
- [ ] [`4.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter6/4.mdx)
- [ ] [`5.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter6/5.mdx)
- [ ] [`6.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter6/6.mdx)
- [ ] [`7.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter6/7.mdx)
- [ ] [`8.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter6/8.mdx)
- [ ] [`9.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter6/9.mdx)
- [ ] [`10.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter6/10.mdx)

**7 - Main NLP tasks**
- [ ] [`1.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter7/1.mdx) [michimichiamo](https://github.com/michimichiamo)
- [ ] [`2.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter7/2.mdx)
- [ ] [`3.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter7/3.mdx)
- [ ] [`4.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter7/4.mdx)
- [ ] [`5.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter7/5.mdx)
- [ ] [`6.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter7/6.mdx)
- [ ] [`7.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter7/7.mdx)
- [ ] [`8.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter7/8.mdx)
- [ ] [`9.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter7/9.mdx)

**8 - How to ask for help**
- [x] [`1.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter8/1.mdx) [EdAbati](https://github.com/EdAbati)
- [x] [`2.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter8/2.mdx)
- [x] [`3.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter8/3.mdx)
- [x] [`4.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter8/4.mdx)
- [x] [`4_tf.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter8/4_tf.mdx)
- [x] [`5.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter8/5.mdx)
- [x] [`6.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter8/6.mdx)
- [x] [`7.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter8/7.mdx)

**Events**
- [ ] [`1.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/event/1.mdx)

**Comment by CaterinaBi** [comment]
Hello Lewis, I'd love to contribute! I'm a postdoctoral researcher in theoretical linguistics at the University of Cambridge, UK. Italian is my native language. I'd love to traslate modules 1 and 2 for a start. The only thing that scares me a bit is that I'm new to GitHub, so I might end up needing some help...

**Comment by lewtun** [comment]
Hey @CaterinaBi thank you - we'd love to have your help with the translation! Feel free to create a post and tag me with @lewtun on our forums (https://discuss.huggingface.co/c/course/20) if you need some help on the GitHub side 🤗

**Comment by CaterinaBi** [comment]
Hey @lewtun, amazing! I'll start straight away. I'll translate the 'Transformer models' and 'Using HF Transformers' then. Do you want me to take care of the Setup instructions, too?

**Comment by lewtun** [comment]
> Hey @lewtun, amazing! I'll start straight away. I'll translate the 'Transformer models' and 'Using HF Transformers' then. Do you want me to take care of the Setup instructions, too?

Awesome! Sorry I tagged you for the Setup section by accident 😬 . On the other hand, that might be an easy way to get familiar with GitHub and pull requests, so maybe you'd like to start there?

**Comment by CaterinaBi** [comment]
Yes, I'll start from there and tag you in the forum if I get lost (get ready to hear from me soon!).

**Comment by sharkovsky** [comment]
Hi @lewtun, I'd also like to contribute. I have a PhD in computational neuroscience from École Polytechnique Fédérale de Lausanne. Italian is my native language as well. 
How about I start with module 3, and see how it goes? Is that acceptable? 

@lewtun, @CaterinaBi, should we think of a way to "standardize" our translations (a shared glossary/vocabulary or something similar)? To make sure we all translate common things such as "train a neural network" in the same way. 

Thank you!

**Comment by ClonedOne** [comment]
Hi! I would also be happy to help. I'm a phd student at Northeastern University and Italian is my native language. 
I can take chapter 4 if no one is working on that.

**Comment by lewtun** [comment]
Hi @ClonedOne thank you for offering to help! I've added your name to Chapter 4 🚀 !

**Comment by Nolanogenn** [comment]
Hi! I would like to help. I am a PhD student at University of Napoli "L'Orientale", and Italian is my native language. I could work on Chapter 5 if nobody's working on it!

**Comment by lewtun** [comment]
Thank you @Nolanogenn for offering to help! I've added your name to the list 🙏

**Comment by sharkovsky** [comment]
@lewtun, @CaterinaBi, @ClonedOne, @Nolanogenn maybe we can come up with a strategy for how to translate common words, for example "machine learning". Here are some options:

1. always leave it in english
2. always translate it in italian (in this case, the official translation is [apprendimento automatico](https://it.wikipedia.org/wiki/Apprendimento_automatico)
3. always translate it in italian, but in the first instance have the english term also associated to it.

The third option looks something like "apprendimento automatic (_machine learning_ in inglese)"

Wikipedia seems to favour option 3, and I would also vote in favour of that. I know that the italian term always sounds a bit "weird", but I feel that since we're making the effort to do a translation anyway, it's nice to try to use as many italian words as possible. 

But I'm open to discussion, what is your opinion?

**Comment by lewtun** [comment]
Thanks for the insight and suggestions into how we can handle the machine learning jargon @sharkovsky ! I really like the analogy with Wikipedia, so would also favour option (3) too. I'm putting together a general `TRANSLATING.md` guide, so will add this suggestion if the other Italian speakers agree it makes sense :)

**Comment by sharkovsky** [comment]
@CaterinaBi, @ClonedOne, @Nolanogenn you can find my first attempt at a translation of one file [in my fork](https://github.com/sharkovsky/course/blob/main/chapters/it/chapter3/1.mdx). I'm happy to receive some feedback if you think some things can be improved/better expressed... I'd rather discuss as much as possible now that we're still in a preliminary phase 😄 

In a provisional manner, I also created a first [glossary ](https://github.com/sharkovsky/course/blob/main/chapters/it/shared_glossary.md) of terms that I think could be useful. But again, I'm happy to discuss both the translations and the format of the glossary! For example, now that I think of it, putting a file in my fork is probably not the best way to share a glossary.... @lewtun do you have any suggestions for something that we could all see and edit?

**Comment by davidemastricci** [comment]
Hi @lewtun, I'm A Data Scientist and a Chatbot Developer and I'd like to help with chapter 6.
I'm attending 🤗HF course and was about to start that chapter, it would be great to translate it while learning.

Italian is my main language!

**Comment by davidemastricci** [comment]
> @lewtun, @CaterinaBi, @ClonedOne, @Nolanogenn maybe we can come up with a strategy for how to translate common words, for example "machine learning". Here are some options:
> 
> 1. always leave it in english
> 2. always translate it in italian (in this case, the official translation is [apprendimento automatico](https://it.wikipedia.org/wiki/Apprendimento_automatico))
> 3. always translate it in italian, but in the first instance have the english term also associated to it.
> 
> The third option looks something like "apprendimento automatic (_machine learning_ in inglese)"
> 
> Wikipedia seems to favour option 3, and I would also vote in favour of that. I know that the italian term always sounds a bit "weird", but I feel that since we're making the effort to do a translation anyway, it's nice to try to use as many italian words as possible.
> 
> But I'm open to discussion, what is your opinion?

@sharkovsky  Since there is a little barrier approaching Hugging Face library, meaning that you should be familiar with terms like Machine Learning and Deep Learning, adding translation that sounds weird in Italian (ex. "apprendimento automatico" or "apprendimento profondo") could make reading less fluent.

**Comment by 1375626371** [comment]
Hello, we are college students studying natural language processing, last month we started the translation of huggingface's course. There was no translation project at the time, so we scraped web pages using crawler, translated them using machine translation, and did the first proofreading by hand. Next, we will carry out the second proofreading and establish a comparison table of professional vocabulary. We want to contribute parts of our translation, can we participate together?

**Comment by lewtun** [comment]
Thanks for this great discussion @sharkovsky - it definitely exposes some subtleties with translation projects :)

For the glossary, I suppose the simplest thing right now would be to share a Google / Notion doc that others can make suggestions to. Notion is probably easier since it supports Markdown and will make it simple to copy back to this repo. 

As for how we distribute the glossary, I see two possibilities:

1. Include it as a standalone file to help guide translators
2. Add it as a new chapter (e.g. at the very end of the course) in an MDX file and render that on the website. 

If you think a glossary would be helpful for course readers, then I would favour option 2.

**Comment by CaterinaBi** [comment]
> Ah, another issue that comes up in italian and may appear in other languages is how you want to address the reader. In english you say: "But what if you want to ...?" In italian you should choose between:
> 
> 1. (informal singular you) "Ma cosa fare se vuoi ....?"
> 2. (informal plural you) "Ma cosa fare se volete ....?"
> 3. (formal singular) "Ma cosa fare se vuole ...?"
> 4. (impersonal) "Ma cosa fare se si vuole ...?"
> 
> Option 4 is equivalent to the english "But what if one wants to ...?"
> 
> I would vote for option 4, except those rare cases where it sounds really clunky and weird, where I would fall back on option 2. But as before, I am open to discuss other ideas!
> 
> You'll probably have the same issue in other languages (french and spanish at least, I assume), so you want to enforce a "centralized" approach through your `TRANSLATING.md` I'll be happy to follow that as well.

Hi guys,

sorry for the late reply but I took a day off yesterday.

I agree with the need to standardise our translations, although I am quite torn when it comes to the question of whether or not we want to translate the technical terms. I believe that if we want a clean Italian version we should use the proposed form 'apprendimento automatico (machine learning)' but at the same time it's true that it's almost a pity to do so while literally anyone in Italy says 'machine learning' (I had to google the translation myself, I wasn't even aware that 'apprendimento automatico' was a thing). So what do we do? @davidemastricci, you had a good point there.

As for the way we address the reader that @sharkovsky mentioned ('But what if you want to ...?') I believe the best translation in Italian would be with an infinitive: 'Ma cosa/come fare per...'. None of the ones that were suggested sound natural to me.

What about the glossary, are we going to add a .mdx file here?

**Comment by CaterinaBi** [comment]
> @CaterinaBi, @ClonedOne, @Nolanogenn you can find my first attempt at a translation of one file [in my fork](https://github.com/sharkovsky/course/blob/main/chapters/it/chapter3/1.mdx). I'm happy to receive some feedback if you think some things can be improved/better expressed... I'd rather discuss as much as possible now that we're still in a preliminary phase 😄
> 
> In a provisional manner, I also created a first [glossary ](https://github.com/sharkovsky/course/blob/main/chapters/it/shared_glossary.md) of terms that I think could be useful. But again, I'm happy to discuss both the translations and the format of the glossary! For example, now that I think of it, putting a file in my fork is probably not the best way to share a glossary.... @lewtun do you have any suggestions for something that we could all see and edit?

Hi @sharkovsky , I've checked out your fork and your first translation seems fine to me ;)

**Comment by sharkovsky** [comment]
Hi everyone, 
following @ClonedOne sensible suggestion, I converted this discussion into a [forum post](https://discuss.huggingface.co/t/standardazing-italian-translation/16352?u=sharkovsky).

@CaterinaBi, @lewtun, @davidemastricci I tried to interpret your votes, but please feel free to correct any mistakes I made. 

Everyone, please let's use the forum post to discuss from now on since it will be much clearer. I will try to monitor it closely and add any words that you suggest to the vocabulary as quickly as possible.

**Comment by davidemastricci** [comment]
@sharkovsky forum link do not work anymore.

**Comment by sharkovsky** [comment]
> @sharkovsky forum link do not work anymore.

yeah it's been marked as spam by the automatic filter, it should be back soon I hope.

**Comment by michimichiamo** [comment]
Hello everyone, I am Michele and recently graduated in Artificial Intelligence at the University of Bologna. Italian is my native language and I would be glad to join the translation! Since Chapter 7 is still to be assigned, I propose to help with that.

**Comment by lewtun** [comment]
Hey @sharkovsky thanks for creating the forum post - I've asked one of the admins to unblock it and hopefully that happens soon 🤞 

@michimichiamo I've added your name to the list - welcome!

**Comment by sharkovsky** [comment]
> Hey @sharkovsky thanks for creating the forum post - I've asked one of the admins to unblock it and hopefully that happens soon 🤞

Thank you, it just got unblocked :D

**Comment by lewtun** [comment]
By the way, I realised from the other translations that we need the first section from Chapter 1 to be translated in order for course to render on the website. My suggestions would be:

- @CaterinaBi would you like to open a pull request with the first section translated?
- Iterate on the glossary in the forum, and then add it as a new "chapter" to the course.

For the second point, we can then have a section in the `_toctree.yml` file with something like:

```
- title: Glossario
  sections:
  - local: glossary/1
    title: Glossario 
```

I think this way Italian readers can benefit from the great work you're doing to handle the various bits of ML jargon!

**Comment by sharkovsky** [comment]
Hi @lewtun, 

just to clarify: we've been calling it a glossary, but it's actually just a vocabulary, i.e. a 1-1 mapping of english and italian terms. Does this meet your expectations? Or did you want us to provide short explanations for each term as well? In any case, I'm happy to oblige! 

@lewtun is there a way to make my forum post editable by anyone, so everyone can contribute to the glossary directly? Otherwise I'm happy to monitor the post and add terms as people suggest them, but of course letting everyone do it for themselves would simplify things. 

To *everyone*: the decision on whether we should translate technical terms such as "machine learning", "training set", "labels", etc... still hasn't been finalized. Ideally, everyone should express an opinion! Please let us know what you think in the forum!

Finally, I've added a new "question" in the post: _should we translate the comments in the code?_

**Comment by lewtun** [comment]
> just to clarify: we've been calling it a glossary, but it's actually just a vocabulary, i.e. a 1-1 mapping of english and italian terms. Does this meet your expectations? Or did you want us to provide short explanations for each term as well? In any case, I'm happy to oblige!

Thanks for clarifying! Since other translators are taking the vocabulary route, I suggest we start with that - we can always expand it to include definitions later if we want to :)



> Finally, I've added a new "question" in the post: _should we translate the comments in the code?_

Thanks! I replied there

**Comment by sharkovsky** [comment]
Sounds great, let's start with a vocabulary and see where we go from there... 

If there is an english source for an "official" glossary containing term definitions and descriptions, I'm sure we can translate that to italian as well!

**Comment by CaterinaBi** [comment]
> By the way, I realised from the other translations that we need the first section from Chapter 1 to be translated in order for course to render on the website. My suggestions would be:
> 
> * @CaterinaBi would you like to open a pull request with the first section translated?
> * Iterate on the glossary in the forum, and then add it as a new "chapter" to the course.
> 
> For the second point, we can then have a section in the `_toctree.yml` file with something like:
> 
> ```
> - title: Glossario
>   sections:
>   - local: glossary/1
>     title: Glossario 
> ```
> 
> I think this way Italian readers can benefit from the great work you're doing to handle the various bits of ML jargon!

Hi @lewtun, I'm sorry I didn't see your message before. I was actually wondering what I was going to do with that first section. You'll have to bear with me because I'm not sure how to do any of these. Let me figure things out, I'll keep you posted.

**Comment by CaterinaBi** [comment]
Hi @lewtun, me again! Just to double check: what you need is the Introduction file, i.e. chapter1/1.mdx, right?

I just have to proof-read it and then I can open a pull request.

However, there are two images in that file that need translating:
1- https://huggingface.co/datasets/huggingface-course/documentation-images/resolve/main/en/chapter1/summary.svg
2- https://huggingface.co/datasets/huggingface-course/documentation-images/resolve/main/en/chapter1/summary-dark.svg

How are we going to take care of these? I can translate them easily, just let me know how to send you the translations.

Thanks!

**Comment by lewtun** [comment]
> Hi @lewtun, me again! Just to double check: what you need is the Introduction file, i.e. chapter1/1.mdx, right?

Yep, that's right! This will allow us to build the website and then we can add the remaining content iteratively :)

About the figures, we don't yet have a good mechanism that allows external contributors to add them to the `documentation-images` repository and doing it manually myself across the 20+ translations we currently have in progress will be too time consuming 🤯 

I suggest we leave the images in English for now, and later add them once we have a means to enable people to contribute directly to `documentation-images`

**Comment by EdAbati** [comment]
Hi everyone,
I just found out about this very nice initiative! 
I am a Data Scientist and Italian is my native language.
I would like to contribute too if you still need some extra help.😀 
What else needs to be done? Maybe I can help with Chapter 8 (I can see that is still unassigned), or is there anything else left?

**Comment by lewtun** [comment]
Hi @EdAbati thank you for offering to contribute! I've added your name to chapter 8 🤗 

Currently we have the first 3 sections of chapter 1 translated, so maybe after chapter 8 it makes sense to discuss with the others to see if they're still actively working on the previous chapters :)

**Comment by CaterinaBi** [comment]
Hi @EdAbati, welcome! I'm working on chapters 1 and 2. @lewtun I've been snowed under with work but I was planning to go back to the translation between tomorrow and tonight. I'll normally be able to do both chapters I promised to translate. Have a great day, guys!

**Comment by sharkovsky** [comment]
Welcome @EdAbati! 
Before starting, please take a quick look at our [discussion](https://discuss.huggingface.co/t/standardazing-italian-translation/16352) on the huggingface forums as well, to try to ensure that we have a consistent translation across chapters. 

@lewtun same here, I apologize but I've been busy with work lately... However I'm still active, just moving very slowly! When people are done with their chapters I'll be happy to offload some of chapter 3, if I haven't finished yet.

**Comment by EdAbati** [comment]
Hi everyone, I hope you are all good!
Not sure if you have seen it but I opened a PR https://github.com/huggingface/course/pull/272 with Chapter 8. It is still WIP I hope to finish soon. In the meantime if you want to proofread it and leave suggestions, feel free :)

**Comment by CaterinaBi** [comment]
Hi guys, I'm not done with my chapters yet but I'll be more than happy to
proofread yours once I'm finished. Caterina

**Comment by sharkovsky** [comment]
Hi all, I've also opened the PR #283 for chapter 3. Everything has been translated but I would like to wait maybe one week to give time to everyone to review it.

**Comment by gdacciaro** [comment]
Hi, I am an Italian student of AI. Can I help you with some chapters?

**Comment by lewtun** [comment]
> Hi, I am an Italian student of AI. Can I help you with some chapters?

Welcome to the group @gdacciaro ! Yes, you are more than welcome to translate some chapters 🚀

One option would be to see if @CaterinaBi is still working on Chapter 2 - if not, perhaps that would be a good place to start?

**Comment by CaterinaBi** [comment]
Hello! He can definitely have chapter 2, I'm afraid I haven't started it
yet!

**Comment by gdacciaro** [comment]
Fine, I will translate chapter 2 :)

**Comment by lewtun** [comment]
> Hi @lewtun, I think I can still help a bit. Are there any other sections that need to be translated?

Nice, thanks for offering to help @EdAbati ! If @gdacciaro or @sharkovsky aren't working on Chapters 2 or 3, I think those would be great to have translated :)

**Comment by EdAbati** [comment]
Hi @lewtun :) I think that Chapter 3 has been translated already in https://github.com/huggingface/course/pull/283

**Comment by CaterinaBi** [comment]
Hi guys. I'm personally still working on chapter 1, I have to translate the
final quiz and then I'll be done!

**Comment by lewtun** [comment]
> Hi @lewtun :) I think that Chapter 3 has been translated already in #283

Oh you're right! I've just updated the issue :)

In that case, Chapter 9 is a safe bet since it was added after this issue was created :)

**Comment by CaterinaBi** [comment]
Lovely! Thanks for your help! Much appreciated!

**Comment by CaterinaBi** [comment]
Hi, I should be translating chapter 2 but I've been snowed under with work
in the last months. If @lewtun is ok with this, you can have it! Caterina

**Comment by EdAbati** [comment]
Hey @sharkovsky , do you want to help me with Chapter 9? I only did the first 3 sections, we can split the remaining 6 if you want.
Do you want to do the last 3 sections: "intro to Blocks", "Gradio check" and "end of chapter quiz"? :)

**Comment by nickprock** [comment]
Hi @gdacciaro , in "Using 🤗 Transformers", "Behind the pipeline" I find a little error.

"Now if we look at the shape of our outputs, the dimensionality will be much lower" is translated as "Ora, se osserviamo la forma dei nostri input, la dimensionalità sarà molto più bassa".

Thanks for your work guys 👍

**Comment by sharkovsky** [comment]
> 

Hi @EdAbati , yes I can do those last three sections! 
Do you mind sharing the link to your fork/branch, so I can take a quick look at what you've already done and try to start from there maybe (if you don't mind! It's not a requirement..)
Thank you!

**Comment by gdacciaro** [comment]
> Hi @gdacciaro , in "Using 🤗 Transformers", "Behind the pipeline" I find a little error.
> 
> "Now if we look at the shape of our outputs, the dimensionality will be much lower" is translated as "Ora, se osserviamo la forma dei nostri input, la dimensionalità sarà molto più bassa".
> 
> Thanks for your work guys 👍

Thanks!

**Comment by EdAbati** [comment]
@sharkovsky , I have already merged the branch with the first 3 sections https://github.com/huggingface/course/pull/373. I will start sections 4-6 in the next days :)

**Comment by nickprock** [comment]
Hi @CaterinaBi @michimichiamo can I help you on chapter 7?
I've already worked on the italian translation of the transformers library documentation.

**Comment by AlessandroMiola** [comment]
Hi everyone:)
May I help with chapter 6?

**Comment by gdacciaro** [comment]
Can I propose a Telegram group to organize the work?

Write me an email (acciarogennaro@gmail.com) and I will send you the link.
